### PR TITLE
Enable dfid_transition in staging/production

### PR DESCRIPTION
**Pull request by tuzz**

### CODE REVIEW

File: hieradata/integration.yaml [hunk 1]
```diff
@@ -23,8 +23,6 @@ govuk::apps::publicapi::backdrop_host: 'www.preview.performance.service.gov.uk'
 govuk::apps::rummager::aws_s3_bucket_name: govuk-api-elasticsearch-snapshots-integration
 
 govuk::apps::smartanswers::expose_govspeak: true
-govuk::apps::dfid_transition::enabled: true
-govuk::apps::dfid_transition::enable_procfile_worker: true
```
Comment (rgarner) [inline hieradata/integration.yaml]: Not sure I'd want to remove these yet; if there are any problems this is the least contentious available production-like environment in which to conduct tests. Can we leave this for a post-live PR?

Comment (tuzz) [inline hieradata/integration.yaml]: dfid_transition will still be enabled in integration. I've set these to `true` in common.yml which this environment inherits from.

Comment (rgarner) [inline hieradata/integration.yaml]: Ignore me. `common.yaml`.

**Comment by rgarner** [comment]
LGTM :+1:

**Pull request merged**